sysdeps/managarm/generic/socket.cpp

```diff
@@ -311,4 +311,35 @@ int sys_setsockopt(int fd, int layer, int number,
 	}
 }
 
+int sys_listen(int fd, int backlog) {
```

Comment:
This probably generates a warning about an unused parameter. Either delete the name `backlog` or do `(void)backlog;` in the body to suppress this.